### DISCUSSION TITLE
<starter assignment not importing>

### DISCUSSION THREAD

**Discussion by jrglass6** [community/community#44348] [GitHub Education]
### Select Topic Area

Bug

### Body

Hello, 
I'm trying to import the starter assignment into my GitHub Classroom but there is an issue with the import I can't resolve. Please help!
<img width="425" alt="Screen Shot 2023-01-14 at 11 13 22 AM" src="https://user-images.githubusercontent.com/22840060/212494689-f2067d45-863a-49a8-9baa-f8ab0ed702ea.png">

**Comment by mhscott** [comment]
I have the same issue.

**Comment by mhscott** [comment]
@jrglass6 I found a workaround (in case you are in a hurry to get a classroom set up like me :) )

- Go to the repo: https://github.com/education/github-starter-course
- Click the green button "Use this template", then select "Create a new repository"
- Create the new repo in your account or organization
- From your classroom, create a new assignment using this repo (You'll be prompted to change the repo settings to "Template")

I'm not a GitHub guru, so likely there's better workarounds until GitHub's one-click workflow is fixed.

**Reply by jrglass6** [reply]
Thank you so much!! This worked for me!
